swatch-contracts/src/test/java/com/redhat/swatch/contract/security/RolesAugmentorTest.java

```diff
@@ -42,15 +42,17 @@ private QuarkusSecurityIdentity securityIdentityForRhIdentityJson(String json) {
     }
   }
 
+  private QuarkusSecurityIdentity securityIdentityForAnonymous() {
+    return QuarkusSecurityIdentity.builder().setAnonymous(true).build();
+  }
+
   @Test
   void testRoleGrantedToRhIdentityWhenEnabled() {
     var augmentor = new RolesAugmentor();
     augmentor.testApisEnabled = true;
-    SecurityIdentity x509Identity =
```

Comment:
This test is for "testRoleGrantedToRhIdentityWhenEnabled".  It seems it should be left with the x509Identity.  But expect both the test and service roles have been added?

Reply:
I misread the predicate that adds the test role (I missed the `!`).  I've corrected the test so both roles are granted now.